packages/opc-feedback/src/opc-feedback.scss

```diff
@@ -30,13 +30,17 @@ $variables: ();
   --opc-feedback--TextColor: #0066CC;
   --opc-feedback__submit--BackgroundColor: #0066CC1A;
   --opc-feedback__submit--TextColor: #0066CC;
+  --opc-feedback__header--BackgroundColor: #0066CC1A;
+  --opc-feedback__header--TextColor: #0066CC;
 }
 
 :host([theme="red"]) {
   --opc-feedback--BackgroundColor: #a30000;
   --opc-feedback--TextColor: #fff;
```

Comment:
```suggestion
  --opc-feedback--TextColor: $opc-color--white;
```

Reply:
can you use the colors already defined in the opc-style

Reply:
updated this